src/rrdp-iostat/rrdp_iostat.ml

```diff
@@ -519,18 +519,39 @@ module Stats_value = struct
     | Some s3 ->
       let last_s3 = last_stats_blktap3 in
       let opt f x = match x with None -> 0L | Some x' -> f x' in
```

Comment:
This an Option.fold, can we delete this line as well?

Reply:
the code would be a lot more verbose if we deleted this helper, but i'll redefine it using fold